app/models/contract.rb

```diff
@@ -0,0 +1,4 @@
+class Contract < ApplicationRecord
```

Comment:
discussed in person: double checking contract as we opted to leave it out initially from MVP as it is not needed, will confirm shortly

Reply:
I don't think we should create `Contract` in the PR as we have yet to decide how we'll migrate `NPQContract`.
I think its only needed for validation `contract_for_cohort_and_course: true`, if we ignore that for now, we can remove contracts.